widget/contrib/gpmdp.lua

```diff
@@ -6,100 +6,97 @@
                                                      
 --]]
 
-local helpers             = require("lain.helpers")
-local json                = require("lain.util.dkjson")
-local watch               = require("awful.widget").watch
-local focused             = require("awful.screen").focused
-local naughty             = require("naughty")
-local next, getenv, table = next, os.getenv, table
+local naughty = require("naughty")
```

Comment:
Yes, Lua does see the libraries already, but specifying them locally avoids more expensive global calls. This is the theory, at least.

And sorry,  last line was a +. Fixed.